scipy/stats/tests/test_distributions.py

```diff
@@ -3682,6 +3682,37 @@ def test_ppf(self, p, a, b, expected):
         x = stats.betaprime.ppf(p, a, b)
         assert_allclose(x, expected, rtol=1e-14)
 
+    def test_ppf_boundary(self):
+        # p == 0 and p == 1.0 need to map to 0.0 and inf
+        # for p == 1.0, division by zero must not occur
+        assert_equal(stats.betaprime.ppf(0.0, 1.3, 0.1), 0.0)
+        assert_equal(stats.betaprime.ppf(1.0, 1.3, 0.1), np.inf)
+
```

Comment:
```suggestion
```
This is assured by the distribution infrastructure.

Reply:
removed